generator/internal/scaffold_generator.cc

```diff
@@ -133,6 +133,12 @@ std::map<std::string, std::string> ScaffoldVars(
     vars.emplace("nameInServiceConfig", api.value("nameInServiceConfig", ""));
     vars.emplace("configFile", api.value("configFile", ""));
   }
+  if (!service.override_service_config_yaml_directory().empty()) {
+    vars.emplace("directory", service.override_service_config_yaml_directory());
```

Comment:
Is "directory" a specific enough name?

Reply:
No.  That motivated a larger cleanup, you may want to take another look.  I will go ahead and merge. If you find any problems we can fix them in a follow up PR.